patterns/template-portfolio-home.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Title: Portfolio Template
+ * Slug: twentytwentyfour/template-portfolio-home
+ * Template Types: front-page, index
+ * Viewport width: 1400
+ * Inserter: no
```

Comment:
I like the flexibility of building with patterns that this little thing brings, lovely